Kaguya/Kaguya/Global.cs

```diff
@@ -17,9 +17,20 @@ public static class Global
         public const string InviteUrl = "https://discord.com/oauth2/authorize?client_id=538910393918160916&scope=bot&permissions=536341759";
         public static readonly string Version = "v4.0-beta-" + Process.GetCurrentProcess().StartTime.ToShortDateString().Replace('/', '.');
 
+        public static int ShardsLoggedIn { get; private set; }
```

Comment:
There is a small bug with this. If a shard needs to login more than once for whatever reason, this number will be incremented again. We don't want that. We instead want a list of shard IDs that have been logged in.